frontend/src/employees/UpdateEmployee.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import EmployeeForm from './EmployeeForm';
+import { putUser, getUser } from '../UserAPI';
+import { useParams } from 'react-router-dom';
+
+function UpdateEmployee() {
+  const [sent, setSent] = useState(false);
+  const [sentSuccessfully, setSentSuccessfully] = useState(false);
+  const [popUpMessage, setPopUpMessage] = useState('');
+  const { id } = useParams();
+  const [userData, setUserData] = useState({});
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const response = await getUser(id);
+        setUserData({ ...response.data, loaded: true });
+      } catch (error) {
+        setUserData({ loaded: true });
+        setPopUpMessage('A módosítás sikertelen');
+        setSent(true);
+      }
+    }
+    fetchData();
+  }, [id]);
+
+  async function putData(values) {
+    try {
+      const response = await putUser(id, values);
+      if (response.status === 200) {
+        setPopUpMessage('Munkavállaló sikeresen módosítva');
+        setSentSuccessfully(true);
+      } else {
+        setPopUpMessage('A módosítás sikertelen');
+      }
+    } catch (error) {
+      setPopUpMessage('A módosítás sikertelen');
+    }
+    setSent(true);
+  }
+
+  return (
+    <>
+      {userData.loaded &&
+        <EmployeeForm
+          sent={sent}
+          setSent={setSent}
+          sentSuccessfully={sentSuccessfully}
+          popUpMessage={popUpMessage}
+          sendData={putData}
+          path='update'
+          title='Adatok módosítása'
+          name={userData.name}
```

Comment:
Alapvetően működik így is, de ha már van egy objektumunk, ami összefogja az összetartozó adatokat, akkor egyszerűbb azt átadni ilyen esetben: user={userData}
Gondolj arra, ha bővül, akkor annyi új prop-od lesz...